guides_and_docs/machine_learning.md

```diff
@@ -0,0 +1,12 @@
+New to Machine leanring hpc using:
+
+1. The hpc resources in UiO (highly recommended):
+https://www.uio.no/tjenester/it/forskning/kompetansehuber/uio-ai-hub-node-project/it-resources/ml-nodes/ 
```

Comment:
Hot tip: you may want to rename your file machine_learning.md and write in markdown :) then you can use [link text](url) to show links